api_factory/templates/$namespace/$name_$version/$service/transports/grpc.py.j2

```diff
@@ -103,8 +103,8 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
         if '{{ method.name|snake_case }}' not in self._stubs:
```

Comment:
I didn't realize we were doing this. What case does this cover?

Reply:
It just covers not creating them all at instance instantiation. I wrote this before getting Thea's feedback in gapic-generator, and should revisit it.

Reply:
Ack.